app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Android Project Template</string>
+    <string name="welcome_fragment_info_text">Welcome to conference app, enter conference ID or click on QR code icon to scan conferences QR code.</string>
```

Comment:
E, ovo za stringove... Ima nešto što mislim da nisi imao prilike kod nas učiti. Kaj je ideja.
Recimo da imaš string "Hello world from bla bla", u tom slučaju napraviš string resurs koji ima ključ baš `hello_world_from_bla_bla` i vrijednost toga bude "Hello world from bla bla", znači:

`<string name="hello_world_from_bla_bla">Hello world from bla bla</string>`

E, i onda kad taj string želiš negdje iskoristiti, napraviš:
`<string name="welcome_fragment_info_text">@string/hello_world_from_bla_bla</string>`

Ideja je kada ćeš eventualno prevoditi apliakciju, puno je lakše znati što neki tekst na rumunjskom znači ako je ključ resursa "hello_world_from_bla_bla"